Doc/library/os.rst

```diff
@@ -1180,8 +1180,27 @@ as internal buffering of data.
    respectively.
 
    .. versionadded:: 3.3
-      Some operating systems could support additional values, like
-      :const:`os.SEEK_HOLE` or :const:`os.SEEK_DATA`.
```

Comment:
Oh, we should do something with this ``versionadded`` block -- probably just remove it?

Reply:
Hm, I did a `git tag --contains 2b47f0a23ffcb0cd86dc9eacf379fbd329b895c7`, and it listed 3.1. See 2b47f0a23ffcb0cd86dc9eacf379fbd329b895c7

Reply:
It is at the end of this list: https://docs.python.org/3/whatsnew/3.3.html#os, which is where I found it from -- very odd. Perhaps it was backported? I'm not sure it matters much, though, given how long ago it was.

Reply:
```console
$ git show 3.1:Modules/_io/bufferedio.c | grep SEEK_HOLE
$ git show 3.2:Modules/_io/bufferedio.c | grep SEEK_HOLE
$ git show 3.3:Modules/_io/bufferedio.c | grep SEEK_HOLE
#ifdef SEEK_HOLE
        && (whence != SEEK_HOLE)
       SEEK_HOLE/SEEK_DATA. */
```

3.3 is it!

Reply:
Anyway, we also need to remove this `versionadded`; it does not make any sense anymore.

Reply:
Actually, let's keep this passage. It makes sense now that we've documented the HOLE/DATA constants.

Reply:
Added back in fa3ee1d44e84c22a8a7cccb59284e217373842d3